### PR TITLE
Added GCE end-to-end test

### PR DESCRIPTION
**Pull request by SaviorAB**
Added test for GCE account:

Motivation

To test if GCE account is working fine
Test will be included into weekly regression via Jenkins job

Specifications

Expected input parmeters:
GCE_PROVIDER_IDENTITY
GCE_PROVIDER_CREDENTIAL
PROVIDER=GCE Account

### CODE REVIEW

File: Berksfile [hunk 1]
```diff
@@ -5,6 +5,7 @@ cookbook 'git', git: "https://github.com/opscode-cookbooks/git.git", ref: "ea7de
 cookbook 'openssl', git: "https://github.com/opscode-cookbooks/openssl.git", ref: "b9a52cdac76063059ff330e8427758cdeff5ee58"
 cookbook 'build-essential', git: "https://github.com/opscode-cookbooks/build-essential.git", ref: "c1f1166bcd20ac4b3dbda518a794936ac4401d7f"
 cookbook 'tomcat', git: "https://github.com/qubell-bazaar/tomcat.git", ref: "1f71ce38431fa3d470d77c1c55ba6902de0bcaf2"
-cookbook 'cookbook-qubell-tomcat', git: "https://github.com/qubell-bazaar/cookbook-qubell-tomcat.git"
-cookbook 'cookbook-qubell-build', git: "https://github.com/qubell-bazaar/cookbook-qubell-build.git"
-cookbook 'cookbook-qubell-broadleaf', git: "https://github.com/qubell-bazaar/cookbook-qubell-broadleaf.git"
+cookbook "windows", git: "https://github.com/opscode-cookbooks/windows.git", ref: "v1.34.6"
```
Comment (dornatsky) [inline Berksfile:8]: why is that needed? is it a part of the test suite or the manifest itself?

File: README.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
 broadleaf-starter-kit
 ===================
 
-Version 1.1-34p
+Version 1.3-39p
```
Comment (dornatsky) [inline README.md:4]: can't be 39. We've only released GCE as part of what, 46?

File: test/test_gce.py [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+import requests
+from qubell.api.globals import DEFAULT_CREDENTIAL_SERVICE, DEFAULT_WORKFLOW_SERVICE
+from qubell.api.private.service import CLOUD_ACCOUNT_TYPE, GCE_CLOUD_TYPE
+
+from qubell.api.testing import *
+from qubell.api.tools import retry
+from testtools import skip
+
+
+def eventually(*exceptions):
```
Comment (dornatsky) [inline test/test_gce.py:11]: don't we have this in the framework already?

File: Berksfile (hunk 1 above)
Comment (SaviorAB) [inline Berksfile:8]: @kgusakov Kirill, could you, please, explain cookbook changes in Berksfile?

File: test/test_gce.py (hunk 3 above)
Comment (SaviorAB) [inline test/test_gce.py:11]: Don't think so, may be in a future plans

Comment (dornatsky) [inline test/test_gce.py:11]: @vasichkin
